ssl/ssl_cipher.cc

```diff
@@ -1446,6 +1446,17 @@ const SSL_CIPHER *SSL_get_cipher_by_value(uint16_t value) {
       ssl_cipher_id_cmp));
 }
 
+const SSL_CIPHER *SSL_CIPHER_find(SSL *ssl, const unsigned char *ptr) {
+  if (ptr) {
+    // Taking first element and left shit by 8 bits to set upper 8 bits.
+    // Bitwise OR with second element to set lower 8 bits of 16 bit integer.
+    uint16_t cipher_id = static_cast<uint16_t>((ptr[0] << 8) | ptr[1]);;
```

Comment:
This just looks a little weird/not common for AWS-LC, can this use one of your new loading functions?

Reply:
yes we can, made the fix